smartcosmos-framework/src/main/java/net/smartcosmos/annotation/SmartCosmosAnonymousUser.java

```diff
@@ -0,0 +1,46 @@
+package net.smartcosmos.annotation;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+
+import net.smartcosmos.security.user.SmartCosmosUser;
+
+/**
+ * Constant interface for ANONYMOUS User configuration.
+ */
+public interface SmartCosmosAnonymousUser {
```

Comment:
Why an interface?  This could be accomplished by having the ANONYMOUS user be a public static on the SmartCosmosUser class.